time.cabal

```diff
@@ -151,3 +150,66 @@ test-suite tests
         Test.AddDays
         Test.AddDaysRef
         Test.TestUtil
+        
+benchmark bench
+    hs-source-dirs: lib, benchmark
```

Comment:
Another approach i can think of is to remove all these stanze, and make a seperate benchmark cabal project in benchmark folder.